src/devtoolsPanel.test.ts

```diff
@@ -467,18 +467,18 @@ describe("devtoolsPanel", () => {
                 expect(mockVsCode.window.showTextDocument).toHaveBeenCalled();
             });
 
-            it("calls getApprovedTabs", async () => {
+            it("calls getNetworkSetting", async () => {

```

Comment:
why is approvedTabs getting renamed to networkSettings, my understanding is that one will retrieve the list of tabs that are allowed to be displayed while the other one seems to retrieve specific settings for network.

Reply:
Talked offline:
- creating new pattern of `get{SETTINGS_NAME}Setting` getter functions for functions that check the vscode settings for the extension
- Landed on `getNetworkSetting`.  The `getApprovedTabs` function would check whether the network pane was enabled/disabled in the vscode settings, so the `getNetworkSetting` name makes sense.